src/screens/OrganizationPeople/AddMember.tsx

```
@@ -297,6 +309,78 @@ function AddMember(): JSX.Element {
+          first: PAGE_SIZE,
+          after: null,
+          last: null,
+          before: null,
+        },
+      });
+    }
+  }, [currentUrl, addUserModalisOpen]);
+
+  // Process data when it arrives
+  useEffect(() => {
+    if (userData && userData.allUsers) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)